src/storm-cli-utilities/model-handling.h

```diff
@@ -1110,7 +1169,7 @@ void verifyWithSparseEngine(std::shared_ptr<storm::models::ModelBase> const& mod
         std::unique_ptr<storm::modelchecker::CheckResult> filter;
         if (filterForInitialStates) {
             filter = std::make_unique<storm::modelchecker::ExplicitQualitativeCheckResult>(sparseModel->getInitialStates());
-        } else {
+        } else if (!states->isTrueFormula()) {
             filter = storm::api::verifyWithSparseEngine<ValueType>(mpi.env, sparseModel, storm::api::createTask<ValueType>(states, false));
```

Comment:
do we need a new else case?  looks like filter is otherwise unitialized? Or are those semantics ok?

Reply:
The next lines say
```c++
        if (result && filter) {
            result->filter(filter->asQualitativeCheckResult());
        }
```
So if `filter` remains uninitialized, no filtering is applied which is exactly what we want if we filter for states satisfying `true`. Maybe I can add `// No need to apply filter for formula 'true'` to make this case more explicit

Reply:
Hi, yeah this is the problem with only reviewing a code snippet :-) The comment does help though.